script/Arcueid/Day 7/07_00_ARC07_1D.txt

```diff
@@ -72,22 +71,23 @@ I have nothing to do with those incidents anymore.
 [7ca172afd97da06366af3e09f1fb1a6decdde09a]{
 -- Page 116, Offset 16865.
 -- 俺はヴローヴと決着をつけた時点で日常に戻ってきた。
-My life returned back to normal the moment I settled the score with Vlov.
+My life returned to normal the moment I settled the score with Vlov.
 }
 [0818345502e0115dfb44b7518b37aaa038d70244]{
 -- Page 116, Offset 16866.
 -- 責任は果たした。果たした筈だ。
-I fulfilled my duty. I should have.
+// 果たした筈だ。 Not sure how to word in English properly
+I fulfilled my duty. At least I should have.
```

Comment:
with this sentence pattern, I'm pretty sure he's affirming himself by saying its bound to be true rather than doubting himself. Open to interpretation though
> I fulfilled my duty. I must have.